app/files.js

```diff
@@ -261,6 +273,43 @@ const folderCreated = {
   }
 }
 
+const folderDeleted = {
+  key: "folderDeleted",
+  noun: "Folders",
+  display: {
+    label: "Folder Deleted",
+    description: "Triggers when a folder is deleted."
+  },
+  operation: {
+    inputFields: [
+      {
+        label: "Room",
+        key: "id",
+        dynamic: "roomCreated.id.title",
+        helpText: "Trigger after deleted from a specific room"
+      }
+    ],
+    /**
+     * @param {ZObject} z
+     * @param {Bundle<SessionAuthenticationData, FolderDeletedFields>} bundle
+     * @returns {Promise<FolderData[]>}
+     */
+    async perform(z, bundle) {
+      const client = new Client(bundle.authData.baseUrl, z.request)
+      const files = new FilesService(client)
+      const filters = {
+        sortBy: "DateAndTime",
+        sortOrder: "descending",
+        filterType: "FoldersOnly"
+      }
+      const trash = await files.listTrash(filters)
+      if (bundle.inputData.id) return trash.folders.filter(item => item.originRoomId === bundle.inputData.id)
```

Comment:
Please avoid omitting the curly braces for function parameters. To help with this, I suggest adding an eslint rule to enforce this guideline.

Reply:
> Please avoid omitting the curly braces for function parameters. To help with this, I suggest adding an eslint rule to enforce this guideline.

Corrected. The eslint rule will be added in a separate PR.